wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,28 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
 
 ## Further exchange and discussion
 
-Sometimes, a single mod and response isn't enough to bring all parties to agreement, or fully resolve an issue. Perhaps the mapper doesn't agree with a suggestion, and the modder isn't satisfied with their response. Alternatively, both parties might agree that although one option is better, it isn't the best option and could still be improved further with additional input. Continued feedback is the best way to improve a beatmap, so get people involved and talking to get as many different ideas, or see things from as many points of view as necessary.
-
-- If you're unsure about something, ask the person in question to explain what they meant. They should be more than willing to explain things to you in more detail.
-- Still can't reach an agreement? Ask someone with more experience, such as a member of [BN](/wiki/People/Beatmap_Nominators) or [NAT](/wiki/People/Nomination_Assessment_Team). Remember they are busy people, however! Should you realise there's a more suitable alternative to something you originally suggested, don't be afraid to put your pride aside - **sometimes being wrong can be far more valuable than being right**.
-- **There is no such things as a "wrong" opinion** — it's up to the mapper and modders to come to an agreement in a civil manner. Try to understand alternative points of view rather than attack them, as you never know when it might be useful in the future.
-- If you can't explain something through logic, you will not be able to explain it to them through violence either. Rather than continue a discussion that isn't progressing, walk away and try again with the next modder/mapper.
-- When things look to be getting out of hand, or if the discussion is struggling to stay on topic, [call a moderator](/wiki/Reporting_bad_behaviour). Don't contribute to the problem by continuing to post.
-
-## Qualified beatmaps
-
-The [Qualified section](/wiki/Beatmap/Category#qualified) allows beatmaps to receive any final suggestions for change or improvement before reaching ranked status. When modding a qualified map, everything you consider "not minor" should **always** be mentioned — minor issues are also worth mentioning however, especially if they affect gameplay. If a qualified beatmap accumulates a significant number of minor issues, or if there is one or more major/unrankable issues, the beatmap should be disqualified so that fixes and improvements can be made before attempting requalification.
+Sometimes, discussions will be drawn-out, and a single mod and response isn't enough to bring all parties to agreement. Perhaps the mapper doesn't agree with a suggestion, and the modder isn't satisfied with their response. Ideally, all discussions should find closure.
```

Comment:
I think this part can add the part to avoid having the discussion that's too long without finding any closure.

something like "Ideally, all discussions should find closure within reasonable amount of time.

Reply:
https://github.com/ppy/osu-wiki/pull/12210/commits/53eee0be91ce576f41a0eaba389ee300b836c7bc The latest change should comply with this suggestion.